### ISSUE TITLE
Feat: Support an Optional Prefix to Use for Redis Keys

### ISSUE THREAD

**Issue by zicklag** [jafayer/dinodns-redis-storage#2]
I'm mixing some other configuration into my redis database in addition to domain records, so it could be handy to be able to set a prefix to have the storage adapter use when storing it's data in Redis.

That said, I might just write my own quick adapter right now because I might have some custom needs, so take this as a suggestion and you don't need to do it just for me. :)

**Comment by jafayer** [comment]
Good call! In the meantime, you should be able to make minimal modifications to the existing plugin by simply overriding the `nameToKey` method. This is a very simple method that centralizes the logic for key generation used in setting and getting the records, so overriding that method should replace the key names for all the other methods appropriately.

Something like this should do the trick

```ts
const store = new RedisStore({...})

store.nameToKey = (name: string) => {
  return `prefix:${name}`;
}
```
